demo/vl-tabs-active-tab.html

```diff
@@ -28,9 +28,13 @@
         <vl-tabs-pane data-vl-id="fiets" data-vl-title="Fiets">
           Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit. Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Praesent commodo cursus magna, vel scelerisque nisl consectetur et.
         </vl-tabs-pane>
+        <vl-tabs-pane data-vl-id="auto">
+          <span slot="auto-title-slot"><b>Auto</b></span>
+          Duis vitae magna vitae eros pretium porttitor id quis mauris. Sed imperdiet a diam in suscipit. Nunc consectetur urna nunc, eu tempor odio rutrum non. Sed in sem convallis, placerat nisi nec, placerat velit. In fringilla ex sed malesuada dictum. Sed congue neque orci, quis porta mi tempus ultrices. Vivamus gravida magna eu enim aliquet, sit amet tempus ante pellentesque. Donec eget hendrerit odio, eget aliquam felis. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras vel maximus orci.
+        </vl-tabs-pane>
```

Comment:
Denk dat de gewoonte is om per "feature" een demo component te maken, dus ik zou hier overwegen om een aparte "vl-tabs met titles als slots" te maken.

Reply:
- nieuwe demo pagina toegevoegd
- vl-tabs-pane verwijdert